lib/src/select/Select.stories.tsx

```diff
@@ -0,0 +1,543 @@
+    <ExampleContainer>
+      <Title title="Value with elipsis" theme="light" level={4} />
+      <DxcSelect label="Value with elipsis" options={optionsWithElipsisLarge} value="1" />
+    </ExampleContainer>
+    <>
+      <Title title="Sizes" theme="light" level={2} />
+      <ExampleContainer>
+        <Title title="Small size" theme="light" level={4} />
+        <DxcSelect label="Small" options={single_options} size="small" />
+      </ExampleContainer>
+      <ExampleContainer>
+        <Title title="Medium size" theme="light" level={4} />
+        <DxcSelect label="Medium" options={single_options} size="medium" />
+      </ExampleContainer>
+      <ExampleContainer>
+        <Title title="Large size" theme="light" level={4} />
+        <DxcSelect label="Large" options={single_options} size="large" />
+      </ExampleContainer>
+    </>
+    <>
+      <Title title="Margins" theme="light" level={2} />
+      <ExampleContainer>
+        <Title title="xxsmall margin" theme="light" level={4} />
+        <DxcSelect label="xxSmall" options={single_options} margin="xxsmall" />
+      </ExampleContainer>
+      <ExampleContainer>
+        <Title title="xsmall margin" theme="light" level={4} />
+        <DxcSelect label="xSmall" options={single_options} margin="xsmall" />
+      </ExampleContainer>
+      <ExampleContainer>
+        <Title title="small margin" theme="light" level={4} />
+        <DxcSelect label="Small" options={single_options} margin="small" />
+      </ExampleContainer>
+      <ExampleContainer>
+        <Title title="medium margin" theme="light" level={4} />
+        <DxcSelect label="Medium" options={single_options} margin="medium" />
+      </ExampleContainer>
+      <ExampleContainer>
+        <Title title="large margin" theme="light" level={4} />
+        <DxcSelect label="Large" options={single_options} margin="large" />
+      </ExampleContainer>
+      <ExampleContainer>
+        <Title title="xlarge margin" theme="light" level={4} />
+        <DxcSelect label="xLarge" options={single_options} margin="xlarge" />
+      </ExampleContainer>
+      <ExampleContainer>
+        <Title title="xxlarge margin" theme="light" level={4} />
+        <DxcSelect label="xxLarge" options={single_options} margin="xxlarge" />
+      </ExampleContainer>
+    </>
+  </>
+);
+
+const OptionsWithElipsisSizeSmall = () => (
+  <div>
+    <DxcSelect label="Small with elipsis" options={optionsWithElipsisSmall} size="small" />
+  </div>
+);
+const OptionsWithElipsisSizeMedium = () => (
+  <div>
+    <DxcSelect label="Small with elipsis" options={optionsWithElipsisMedium} size="medium" />
+  </div>
+);
+const OptionsWithElipsisSizeLarge = () => (
+  <div>
+    <DxcSelect label="Small with elipsis" options={optionsWithElipsisLarge} size="large" />
+  </div>
+);
+const DefaultSelect = () => (
+  <div>
+    <DxcSelect label="Label" options={single_options} placeholder="Choose an option" />
+  </div>
+);
+const DefaultSearchableSelect = () => (
```

Comment:
Should we take advantage of this example to also test clearable hover?

```
const DefaultSearchableSelect = () => (
  <ExampleContainer pseudoState="pseudo-hover">
    <DxcSelect label="Label" searchable options={single_options} placeholder="Choose an option" />
  </ExampleContainer>
);
```

![image](https://user-images.githubusercontent.com/44321109/152777312-13ff47a7-1253-4935-8be7-d19441052909.png)

Reply:
This case will be implemented once we investigate how to do this, an issue has been created to solve this problem.[issue](https://github.com/dxc-technology/halstack-react/issues/629)